### ISSUE TITLE
Switch to new credential IDs in all Jenkinsfiles

### ISSUE THREAD

**Issue by Hejado** [eclipse/xtext#1874]
Tracking: https://bugs.eclipse.org/bugs/show_bug.cgi?id=568365

**Comment by Hejado** [comment]
PRs are merged. Gonna keep this issue open for a bit, at least until the old credentials are removed, so we can double check.

**Comment by Hejado** [comment]
Old credentials are now removed, and I switched the multibranch jobs to the new credentials as well. (which I couldn't do before, because old and new credentials were indistinguishable in the gui config)

Looks like the short time of anonymous polling exceeded the github api, maybe? 

> Jenkins-Imposed API Limiter: Still sleeping, now only 20 min remaining

If that's the case, the problem should fix itself in a few minutes.
